src/components/molecules/InputProfileImage/InputProfileImage.tsx

```diff
@@ -1,25 +1,25 @@
+import React, { useState } from 'react';
 import Label from '@/components/atoms/Label/Label';
 import ProfileSetting from '@/components/atoms/ProfileSetting/ProfileSetting';
 import {
   InputProfileImageProps,
   useCheckProfileImage,
 } from '@/hooks/common/inputsUserInfo/useCheckProfileImage';
-import React, { useState } from 'react';
 import DefaultProfileModal from '../DefaultProfileModal/DefaultProfileModal';
-import {
-  defaultProfileModalcenterStyling,
-  profileDefaultText,
-  profileImageSelectContainer,
-  profileImageText,
-  profileImageTextContainer,
-} from './InputProfileImage.style';
+import * as S from './InputProfileImage.style';
 
 const InputProfileImage = ({
-  setProfilePhoto,
+  setImageFileId,
   imgSrc,
+  setIsImgChanged,

```

Comment:
```suggestion
  setIsImageChanged,
```
이렇게 line 12 과 통일감을 가져가도 좋을 거 같아요!

Reply:
수정했습니다!! [5d5934c](https://github.com/JECT-Study/PICK-O-Client/pull/244/commits/5d5934cb0192186b405e1baf8ea0e73e841ffe0d)